packages/runtime/lib/management-api.js

```diff
@@ -115,7 +117,11 @@ async function createManagementApi (configManager, runtimeApiClient) {
     })
 
     app.get('/logs', { websocket: true }, async (connection) => {
-      process.stdout.pipe(connection.socket)
+      loggingPort.on('message', (message) => {
+        for (const log of message.logs) {
+          connection.socket.send(log)
+        }
+      })
```

Comment:
If the WebSocket goes down, you must remove the event listener. Otherwise, there will be a massive memory leak.